README.md

```diff
@@ -83,6 +83,28 @@ function home() {
 }
 ```
 
+## Log Levels
+
+You can control which log levels are sent to Axiom by setting the `AXIOM_LOG_LEVEL` environment variable. The default is `debug`.
```

Comment:
@bahlo @lukasmalkmus wondering if the default level should be info.

Reply:
debug sounds good 👍